core/resources/src/main/res/values-night/themes.xml

```diff
@@ -14,13 +14,16 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-<resources xmlns:tools="http://schemas.android.com/tools">
+<resources>
 
     <style name="NightAdjusted.Theme.GoCart" parent="android:Theme.Material.NoActionBar" />
 
-    <style name="NightAdjusted.Theme.SplashScreen" parent="Theme.SplashScreen">
-        <item name="android:windowLightStatusBar" tools:targetApi="23">false</item>
-        <item name="android:windowLightNavigationBar" tools:targetApi="27">false</item>
```

Comment:
This style is necessary so that the system bars are visible
![Splash-Dark-systembars](https://github.com/etonotieno/GoCart/assets/25648109/c49b3140-9f92-4b99-9002-eb48dfbc3226)
![Splash-Light-systembars](https://github.com/etonotieno/GoCart/assets/25648109/7c9aebc7-bafc-4613-ba91-f6f1da1803cf)